Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -55,10 +55,11 @@ def tap_new
       jobs:
       - job: macOS
         pool:
-          vmImage: xcode9-macos10.13
+          vmImage: macOS-10.13
```

Comment:
I feel like this change should at least be in a separate commit if not a separate PR. The commit message doesn't explain why this or line 62 changed.

Reply:
I think it could have been included in #5600 but was missed

Reply:
As @scpeters said. Commit message amended.